cli/uninstall/uninstall.go

```diff
@@ -112,12 +112,30 @@ func UninstallProgram(program string, programVersion string, binDst string, head
 			return err
 		}
 	}
+	// We need to try to delete all versions with 'v' prefix before x.y.z version
```

Comment:
This function is already big enough. Let's extract the code, which decides which versions to be remove to a separate function.

Reply:
Extracted to it's own function.